### DISCUSSION TITLE
Keep seeing "Please remove the extra to hide this warning" in notifications

### DISCUSSION THREAD

**Discussion by TheCreatorAMA** [LazyVim/LazyVim#1684] [Q&A] [answered]
I keep seeing "Please remove the extra to hide this warning" in my notifications when I try add options for `conform.nvim` and `nvim-lint`.

In my plugins folder I added the below to extend the options for each plugin

**plugins/conform.lua**
```lua
return {
  {
    "stevearc/conform.nvim",
    opts = {
      formatters_by_ft = {
        lua = { "stylua" },
        python = { "isort", "black" },
        javascript = { { "prettierd", "prettier" } },
        fish = { "fish_indent" },
        sh = { "shfmt" },
      },
    },
  },
}

```

**plugins/nvim-lint.lua**
```lua
return {
  {
    "mfussenegger/nvim-lint",
    opts = {
      -- Event to trigger linters
      events = { "BufWritePost", "BufReadPost", "InsertLeave" },
      linters_by_ft = {
        fish = { "fish" },
        python = { "flake8" },
      },
    },
  },
}
```

How can I get rid of those notifications?

**Comment by folke** [comment, marked as answer]
You still have a reference to the conform/nvim-lint extras in `lua/config/lazy.lua`

**Comment by TheCreatorAMA** [comment]
Was being a dummy and sorted it out... Had to take out the imports for these under spec in `lazy.lua`
